### ISSUE TITLE
should we sometimes be using `FRBRname` instead of `FRBRnumber`?

### ISSUE THREAD

**Issue by goose-life** [laws-africa/cobalt#43]
By-laws aren't always given numbers, so we use their titles for disambiguating them instead.

We might potentially use titles for judgments as well, depending on how reliably they are numbered.

This is correct according to https://docs.oasis-open.org/legaldocml/akn-nc/v1.0/os/akn-nc-v1.0-os.html#_Toc531692270, which says one should use "Number or title or other disambiguating feature of the Work" here.

However, it goes on to say "For an Akoma Ntoso XML representation, this value MUST correspond to the content of element <FRBRnumber> or <FRBRname>, respectively, in the metadata".

Do they mean that if a number isn't used, the element should be `FRBRname`, not `FRBRnumber`?

**Comment by longhotsummer** [comment]
Yes, I think you're correct that we should probably use FRBRname when FRBRnumber isn't strictly a number. I'm not sure how we get cobalt to detect that, since something like 'cap17' is technically a number.

I'm not sure what "required when necessary for disambiguation, optional otherwise" at the end of that sentence in the docs means.

Also, the [docs for FRBRnumber](http://docs.oasis-open.org/legaldocml/akn-core/v1.0/os/part2-specs/os-part2-specs_xsd_Element_FRBRnumber.html) say "FRBRnumber is the metadata property containing a string or number for the number to be used in the work-level IRI of this document", so I've always interpreted this to mean that we can just use FRBRnumber for both.
